eiffel-syntax-and-usage/the-meta-object.md

```diff
@@ -18,7 +18,7 @@
 # The Meta Object
```

Comment:
As stated in the other file we should update this one instead.

Reply:
Done